opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -533,6 +533,11 @@ def _format_links(links):
             f_links.append(f_link)
         return f_links
 
+    @staticmethod
+    def _format_resource(resource):
+        resource_json_obj = json.loads(resource.to_json())
+        return {k: v for k, v in resource_json_obj.items() if v}
```

Comment:
I think we would want to write everything to console whether the value is truthy or not.

Reply:
That would be a breaking change, though, given that the current set up does not include null or empty values. I agree, but just wanted to confirm.

Reply:
Not outputting some kv pairs because value is empty is bug. We should stdout even if the value is empty.